Documentation/building/Running-Tests-In-VisualStudio-IDE.md

```diff
@@ -0,0 +1,19 @@
+# Running unit tests within Visual Studio
+
+Sometimes it is convenient to run individual unit tests within the Visual Studio IDE
+
+There are several environment variables which must be set to get this to work correctly.  To make things easier there is a convenience script `visual-studio-devenv.cmd` provided.  This script will set the necessary environment variables.
+
+## Steps
+
+1 `build.cmd`
+2 `visual-studio-devenv.cmd`
+3 Open the test explorer window within the Visual Studio IDE
+4 Select tests and run and/or debug.
```

Comment:
Use markdown syntax (with a period after each number) to make this list render properly.